console/command.go

```diff
@@ -397,24 +399,31 @@ var getAddressesByAccountCmd = &cobra.Command{
 	},
 }
 var importPriKeyCmd = &cobra.Command{
-	Use:   "importprivkey {priKey} {pripassword}",
+	Use:   "importprivkey {priKey} {pripassword} {raw(hex), wif(Base58); Private Key format; default: raw}",
 	Short: "import priKey ",
 	Example: `
-		importprivkey priKey pripassword
+		importprivkey priKey pripassword ef235aacf90d9f4aadd8c92e4b2562e1d9eb97f0df9ba3b508258739cb013db2
```

Comment:
the example is wrong. should be 
`importprivkey ef235aacf90d9f4aadd8c92e4b2562e1d9eb97f0df9ba3b508258739cb013db2 pripassword`
and 
`importprivkey 5HueCGU8rMjxEXxiPuD5BDku4MkFqeZyd4dZ1jvhTVqvbTLvyTJ pripassword wif`

Reply:
27d8bc668c70da7da271ebd6ea1dee42d2c81212